go/vt/vtgate/schema/tracker.go

```diff
@@ -300,21 +300,11 @@ func (t *Tracker) updateTables(keyspace string, res *sqltypes.Result) {
 
 func (t *Tracker) updatedViewSchema(th *discovery.TabletHealth) bool {
 	viewsUpdated := th.Stats.ViewSchemaChanged
-	views, err := sqltypes.BuildBindVariable(viewsUpdated)
-	if err != nil {
-		log.Errorf("failed to read updated views from TabletHealth: %v", err)
-		return false
-	}
-	bv := map[string]*querypb.BindVariable{"viewNames": views}
-	res, err := th.Conn.Execute(t.ctx, th.Target, mysql.FetchUpdatedViews, bv, 0, 0, nil)
+	res, err := th.Conn.GetSchema(t.ctx, th.Target, querypb.SchemaTableType_VIEWS, viewsUpdated)
 	if err != nil {
 		t.tracked[th.Target.Keyspace].setLoaded(false)
 		// TODO: optimize for the views that got errored out.
 		log.Warningf("error fetching new views definition for %v", viewsUpdated, err)
-		code := vterrors.Code(err)
-		if code == vtrpcpb.Code_UNAUTHENTICATED || code == vtrpcpb.Code_PERMISSION_DENIED {
-			log.Warning(aclErrorMessageLog)
-		}
```

Comment:
What's the reason for removing this?

Reply:
GetSchema does not go through ACL check, so tracker will not see such error message.

Reply:
Got it